src/java.base/share/native/libjava/ClassLoader.c

```diff
@@ -99,7 +99,12 @@ Java_java_lang_ClassLoader_defineClass1(JNIEnv *env,
         return 0;
     }
 
+    // On AIX malloc(0) returns NULL which looks like an out-of-memory condition; so adjust it to malloc(1)
+    #ifdef _AIX
+    body = (jbyte *)malloc(length == 0 ? 1 : length);
```

Comment:
Can we use identification in the ifdef/else/endif block to make it a bit more readable. Also can you trim down the comment or split it over two lines to avoid the really long line (it makes it a bit easier for future side-by-side reviews).

Reply:
I can split down the comment if you prefer. It might be appropriate to do in the as-yet-unmerged cleanup PR I have open for the same file.

I am not sure what you mean by 'use identification'. Can you clarify?

Reply:
Ah, that does make sense. Please request these changes [here](https://github.com/openjdk/jdk/pull/7846) and I am happy to make them.